pkg/kubelet/volume/gce_pd/gce_pd.go

```diff
@@ -196,7 +196,7 @@ func (pd *gcePersistentDisk) SetUp() error {
 	}
 
 	// Perform a bind mount to the full path to allow duplicate mounts of the same PD.
-	globalPDPath := makeGlobalPDName(pd.plugin.host, pd.pdName, pd.readOnly)
+	globalPDPath := makeGlobalPDName(pd.plugin.host, pd.pdName)
```

Comment:
While you're making cleanups, can I suggest one more?

Above, when we call AttachDisk() - rename to AttachAndMountDisk(), and have that accept both the pd and the globalPDPath on which to mount.  This will make the global-related code less coupled to PD (since we want to reuse some of it for iscsi and maybe NFS).